lib/cli/ui/formatter.rb

```diff
@@ -136,6 +137,19 @@ def parse_expr(sc, stack)
               index
             )
           end
+        elsif match = sc.scan(SCAN_WIDGET)
+          match_data = SCAN_WIDGET.match(match)
```

Comment:
This is kind of shitty. I could have used `Regexp.last_match`, but it wouldn't have been threadsafe.